tests/sles4sap/publiccloud/check_ibsm_embargoed_azure.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Checks for embargoed updates on IBSM
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use strict;
+use warnings;
+use base 'sles4sap_publiccloud_basetest';
+use publiccloud::utils "validate_repo";
+use testapi;
+
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
+sub run() {
+    my ($self, $run_args) = @_;
+    my $instance = $run_args->{my_instance};
+    $self->{network_peering_present} = 1 if ($run_args->{network_peering_present});
+    record_info('CONTEXT LOG', "instance:$instance network_peering_present:$self->{network_peering_present}");
+
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+    my $count = 0;
+
+    while (defined(my $maintrepo = shift @repos)) {
+        next if $maintrepo =~ /^\s*$/;
+        if (!validate_repo($maintrepo)) {
+            foreach my $instance (@{$run_args->{instances}}) {
```

Comment:
True. I basically left the loop in the case that we want - for whatever reason, I can't think of any right now - to run it on more machines in the future, since it's just one machine anyway.

I will change it to only use the first element

Reply:
done